### DISCUSSION TITLE
File manager and A-shell/vim interaction

### DISCUSSION THREAD

**Discussion by mtcabeza** [holzschu/a-shell#751] [Q&A]
when i select a .py file in the file manager it opens it in a-shell under vim.  Is there a way to disable this behavior?

**Comment by holzschu** [comment]
Hi, 
This behaviour is from iOS, not from a-Shell: when you click on a file in the file manager, it is opened by any of the apps that can open this file type, randomly (on my iPhone, Python files open in Pythonista). The last time I checked, there was no way to select which app was opened for each file type. Let me check again.

**Reply by holzschu** [reply]
This seems to work on my iPhone: long press on a Python file, a pop-up menu appears. Click on "Share", then scroll laterally to "...More". A menu with apps appears, select the app you want to associate with Python files. The behaviour seems to be persistent: clicking on a different Python file now opens the app I selected. I got the suggestion from here: https://apple.stackexchange.com/questions/405528/files-app-how-to-control-which-app-opens-when-a-file-is-tapped

**Reply by mtcabeza** [reply]
thanks holzschu!  everything works as described. i am using ipadOs 17.3
